examples/js/loaders/OBJLoader.js

```diff
@@ -688,18 +688,14 @@ THREE.OBJLoader = ( function () {
 						if ( isLine && material && ! ( material instanceof THREE.LineBasicMaterial ) ) {
 
 							var materialLine = new THREE.LineBasicMaterial();
-							THREE.Material.prototype.copy.call( materialLine, material );
```

Comment:
Did you modify this file by mistake?

Reply:
When I started my idea was to replace OBJLoader. I reverted in a later commit. Shall I squash parts of the history, so nobody gets confused?

Reply:
Please revert these changes. Besides, I think it's no good idea to replace `OBJLoader`. From my point of view, `OBJLoader2` is not suitable to replace `OBJLoader` for different reasons (e.g. the general code style and architecture of the loader).

Reply:
Sorry for the lack of feedback but the `OBJ` format is not that important compared to other 3D formats these days.

In any event, `OBJLoader2` does things like input validation (the usage of `THREE.LoaderSupport.Validator`). In general, `three.js` does not perform something like that for performance reasons. I also think that the general handling of `LoaderSupport` support feels like over-engineering to me. You clearly overestimate the benefit of workers in context of loading 3D assets. From my point of view we don't need a generic worker solution for loaders. The only valid use case for me is in context of `Draco` decompression right now.